src/ConfigEditor.tsx

```diff
@@ -76,7 +75,7 @@ export class ConfigEditor extends PureComponent<Props, State> {
               labelWidth={8}
               inputWidth={20}
               onChange={this.onURLChange}
-              value={jsonData.scalyrUrl}
+              value={jsonData.scalyrUrl || 'https://app.scalyr.com/'}
```

Comment:
(minor small possible future improvement) We could define this value as part of a constant and reference it everywhere.

Reply:
Agree, will defer this.  Prefer all the frontend code gets a review (will make a ticket for this as well)